tests/images.bats

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "images" {
+  cid1=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  cid2=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json busybox)
+  run buildah --debug=false images
+  [ $(wc -l <<< "$output") -eq 3 ]
+  [ "${status}" -eq 0 ]
+  buildah rm -a
+  buildah rmi -a -f
+}
+
+@test "images filter test" {
+  cid1=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  cid2=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json busybox)
+  run buildah --debug=false images --filter since=alpine
+  [ $(wc -l <<< "$output") -eq 2 ]
+  [ "${status}" -eq 0 ]
+  buildah rm -a
+  buildah rmi -a -f
+}
+
+@test "images format test" {
+  cid1=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  cid2=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json busybox)
+  run buildah --debug=false images --format "{{.Name}}"
+  [ $(wc -l <<< "$output") -eq 1 ]
```

Comment:
My expectation is to output two lines. But I don't understand why only one line was output in the test.

Reply:
Did you fix this?

Reply:
I'm seeing two lines when I run this "by hand", which is the result I'd expect to see.

```
# buildah images --format "{{.Name}}"
docker.io/library/alpine:latest
docker.io/library/busybox:latest
```

Reply:
@rhatdan something funky with this.  This should be "-eq 2" not "-eq 1", I'm not sure why it's passing.